src/acsets/ACSetInterface.jl

```diff
@@ -140,7 +140,7 @@ Returns the ID of the added part.
 
 See also: [`add_parts!`](@ref).
 """
-@inline function add_part!(acs, type::Symbol, kw)
+@inline function add_part!(acs ::ACSet , type::Symbol, kw)
```

Comment:
The usual style in Julia is to not put spaces on either side of `::`, e.g., this should line be `acs::ACSet`.